doc/DEPENDENCIES.md

```diff
@@ -6,15 +6,16 @@
 | ------------- |:------------- | :-----:| :-------: |
 | [Google Test](#google-test) | submodule  | `release-1.10.0` | |
 | [C/C++ Compiler](#c-and-c-compiler) | external | see below |  |
-| [CMake](#cmake) | external | \>= `3.14` | |
+| [CMake](#cmake) | external | \>= `3.15` | |
```

Comment:
Didn't catch this before, but the minimum version required for CMake is 3.17, see the root `CMakeLists.txt`.